sim/paladin/paladin.go

```diff
@@ -48,6 +48,9 @@ const (
 	// Judgements
 	ClassSpellMask_PaladinJudgements = ClassSpellMask_PaladinJudgementOfCommand | ClassSpellMask_PaladinJudgementOfMartyrdom |
 		ClassSpellMask_PaladinJudgementOfRighteousness | ClassSpellMask_PaladinJudgementOfTheCrusader
+
+	ClassSpellMask_PaladinHarmfulGCDSpells = ClassSpellMask_PaladinExorcism | ClassSpellMask_PaladinHolyWrath | ClassSpellMask_PaladinHolyShock |
```

Comment:
The tooltip explicitly *excludes* Holy Shock
![image](https://github.com/user-attachments/assets/17e60cf4-1869-4c90-ad60-076093ac4c6f)

Reply:
I actually included that in the Infusion of Souls code rather than here to keep this more general purpose

![image](https://github.com/user-attachments/assets/4c736340-eb07-414d-93aa-977f76721d12)